deployments/ecmps/release-v2.0-fix/Sprint28/camdecmpswks/AddedConstraints.sql

```diff
@@ -0,0 +1,71 @@
+ALTER TABLE camdecmpswks.monitor_default
+ADD CONSTRAINT uq_monitor_default_key
+UNIQUE (mon_loc_id, parameter_cd, default_purpose_cd, fuel_cd, operating_condition_cd, begin_date, begin_hour);
+
+ALTER TABLE camdecmpswks.monitor_formula
+ADD CONSTRAINT uq_monitor_formula_key
+UNIQUE (mon_loc_id, formula_identifier);
+
+ALTER TABLE camdecmpswks.monitor_load
+ADD CONSTRAINT uq_monitor_load_key
+UNIQUE (MON_LOC_ID, BEGIN_DATE, BEGIN_HOUR);
+
+ALTER TABLE camdecmpswks.monitor_location_attribute
+ADD CONSTRAINT uq_monitor_location_attribute_key
+UNIQUE (MON_LOC_ID, BEGIN_DATE);
+
+ALTER TABLE camdecmpswks.monitor_method
+ADD CONSTRAINT uq_monitor_method_key
+UNIQUE (MON_LOC_ID, PARAMETER_CD, BEGIN_DATE, BEGIN_HOUR);
+
+ALTER TABLE camdecmpswks.monitor_system
+ADD CONSTRAINT uq_monitor_system_key
+UNIQUE (MON_LOC_ID, SYSTEM_IDENTIFIER);
+
+ALTER TABLE camdecmpswks.monitor_system_component
+ADD CONSTRAINT uq_monitor_system_component_key
+UNIQUE (MON_SYS_ID, COMPONENT_ID, BEGIN_DATE, BEGIN_HOUR);
+
+ALTER TABLE camdecmpswks.analyzer_range
+ADD CONSTRAINT uq_analyzer_range_key
+UNIQUE (COMPONENT_ID, BEGIN_DATE, BEGIN_HOUR);
+
+ALTER TABLE camdecmpswks.system_fuel_flow
+ADD CONSTRAINT uq_system_fuel_flow_key
+UNIQUE (MON_SYS_ID, BEGIN_DATE, BEGIN_HOUR);
+
+ALTER TABLE camdecmpswks.monitor_qualification
+ADD CONSTRAINT uq_monitor_qualification_key
+UNIQUE (MON_LOC_ID, QUAL_TYPE_CD, BEGIN_DATE);
+
+ALTER TABLE camdecmpswks.monitor_qualification_pct
+ADD CONSTRAINT uq_monitor_qualification_pct_key
+UNIQUE (MON_LOC_ID, MON_QUAL_ID, QUAL_YEAR);
```

Comment:
This table doesn't have a MON_LOC_ID column.
```suggestion
UNIQUE (MON_QUAL_ID, QUAL_YEAR);
```